docs/content/concepts/testing.mdx

```diff
@@ -193,13 +193,105 @@ def my_foo_resource(context):
 
 
 def test_op_resource_def():
-    context = build_op_context(
-        resources={"foo": my_foo_resource.configured({"my_str": "bar"})}
-    )
+    context = build_op_context(resources={"foo": my_foo_resource.configured({"my_str": "bar"})})
     assert op_requires_foo(context) == "found bar"
 ```
 
-### Testing job execution with configs
+Note that when directly invoking ops, io managers specified on inputs and outputs are not used.
+
+### Testing assets
+
+Similar to ops, <PyObject object="asset" decorator /> decorated functions can be directly invoked. Doing so will invoke the underlying op computation.
+
+A basic asset, with no dependencies:
+
+```python file=/concepts/ops_jobs_graphs/unit_tests.py startafter=start_test_basic_asset endbefore=end_test_basic_asset
+from dagster import asset
+
+
+@asset
+def basic_asset():
+    return 5
+
+
+# An example unit test for basic_asset.
+def test_basic_asset():
+    assert basic_asset() == 5
+```
+
+An asset with input dependencies:
+
+```python file=/concepts/ops_jobs_graphs/unit_tests.py startafter=start_test_input_asset endbefore=end_test_input_asset
+from dagster import asset
+
+
+@asset
+def asset_with_inputs(x, y):
+    return x + y
+
+
+# An example unit test for asset_with_inputs.
+def test_asset_with_inputs():
+    assert asset_with_inputs(5, 6) == 11
+```
+
+Any resources defined on your asset (either directly, or using <PyObject object="with_resources" />) will be picked up when invoking the asset. Simply provide a context using <PyObject object="build_op_context" />.
```

Comment:
```suggestion
Any resources defined on your asset - either directly or using <PyObject object="with_resources" /> - will be picked up when invoking the asset. Simply provide a context using <PyObject object="build_op_context" />.
```

Is using the context required to pick up defined assets? If so, I suggest making this sentence reflect that - right now it sounds like a suggestion.

Reply:
good point - rewording it a bit